package.json

```diff
@@ -11,10 +11,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/jpillora/node-glob-all.git"
+    "url": "https://github.com/sr4001/node-glob-all.git"
```

Comment:
?

Reply:
Sorry, that was unintentionally committed.  Just trying to work on my side on the glob version issue